dozer/cogs/development.py

```diff
@@ -82,6 +83,7 @@ def __local_check(self, ctx):  # All of this cog is only available to devs
 
 
 def load_function(code, globals_, locals_):
+    """Loads functions if possible"""
```

Comment:
This specifically loads the user-evaluted code as a function so it can be executed.